ui/src/settings.vue

```diff
@@ -245,12 +267,13 @@
 import Vue from 'vue'
 import pageheader from '@/components/pageheader'
 import statustag from '@/components/statustag'
+import Password from 'vue-password-strength-meter'
 
 const lib = require('@/lib'); //for avatar_url
 
 export default {
     components: { 
-        pageheader, statustag,
+        pageheader, statustag, Password
```

Comment:
Can you update to use the *new* split loading method for this? Especially for vue-password-strength-meter.